### PR TITLE
reword account settings and make like current manager

### LINKED ISSUE #1390: Account settings review current manager wording

### PR DESCRIPTION
**Pull request by na3d**
Close #1390 
screen shot 2017-05-12 at 11 18 09 am

### CODE REVIEW

**Comment by rmcintosh** [comment]
Not a huge deal but if we could have the hyphens aligned for options in these types of menus, I think it would look better:

screen shot 2017-05-12 at 3 42 17 pm

**Comment by na3d** [comment]
Yeah issue 1711 was created to address that.

**Comment by rmcintosh** [comment]
ah, I see - nice! nvm then

**Comment by abemassry** [comment]
image


image

**Comment by na3d** [comment]
How did you manage that error? hahaha

**Comment by mparke** [comment]
linter errors

File: test/settings/layouts/IndexPage.spec.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ describe('settings/layouts/IndexPage', () => {
       />
     );
 
-    page.find('#networkHelper').simulate('change');
+    const helper = page.find('input[name="networkHelper"]').at(0);
+    helper.simulate('change', { target: {
+      name: 'networkHelper',
+      value: 'true',
+    } });
```
Comment (eatonphil) [inline test/settings/layouts/IndexPage.spec.js]: Does this make sure that the correct networkHelper value is submitted to the api?

Comment (na3d) [inline test/settings/layouts/IndexPage.spec.js]: Yup

**Pull request merged**